docs/en/sql-reference/functions/other-functions.md

```diff
@@ -4390,3 +4390,37 @@ Result:
 1. │ ['{ArraySizes}','{ArrayElements, TupleElement(keys), Regular}','{ArrayElements, TupleElement(values), Regular}'] │
    └──────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
 ```
+
+## globalVariable
+
+Takes constant string argument and returns the value of global variable with that name. It is intended for compatibility with MySQL.
```

Comment:
Could add something like "This function is not useful for normal operation of ClickHouse."

Reply:
Created a new PR: https://github.com/ClickHouse/ClickHouse/pull/70938